src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -84,17 +91,38 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             if (isset($rootClass, $source[$rootProperty = $info->fieldName], $source[ItemNormalizer::ITEM_KEY])) {
                 $rootResolvedFields = $this->identifiersExtractor->getIdentifiersFromItem(unserialize($source[ItemNormalizer::ITEM_KEY]));
                 $subresource = $this->getSubresource($rootClass, $rootResolvedFields, array_keys($rootResolvedFields), $rootProperty, $resourceClass, true, $dataProviderContext);
+
+                if (null !== $this->dispatcher) {
```

Comment:
Can't we use a `voidDispatcher` to avoid all these null checks?

Reply:
Discussed in private with @dunglas and @alanpoulain, reached a consensus: keeping the current form (`null !==`) for the following reasons:
- best performance;
- less code to maintain.